Lab1/lab1_polynomial.h

```diff
@@ -81,5 +81,6 @@ class PolynomialTest{
     bool testEquivalence();
     bool testPrint();
     bool testAddition();
+    bool testSubtraction();
     void run();
 };
```

Comment:
can you add a newline at the end of the header file on your next commit for indexing?

Reply:
will do